src/axom/primal/tests/primal_tetrahedron.cpp

```diff
@@ -623,6 +624,30 @@ TEST_F(TetrahedronTest, regularTetrahedron)
     EXPECT_EQ(primal::ON_POSITIVE_SIDE, primal::orientation(pt, tri));
   }
 }
+
```

Comment:
Nice use of `next_permuation`!

It might be good to have a test w/ some tets that have positive and negative orientations and checking that the absolute value of the volume stays the same before/after calling `checkAndFixOrientation()`, but that the sign is non-negative after calling it.